client/src/components/SignUp.tsx

```
@@ -202,15 +202,19 @@ const SignUp = () => {
+            }
           >
             {showConfirmPassword ? <EyeOff size={20} /> : <Eye size={20} />}
           </button>
-          {errors.confirmPassword && (
+          {errors?.confirmPassword && (
             <span className="text-red-500">
-              {errors.confirmPassword &&
-                errors.confirmPassword.message &&
-                String(errors.confirmPassword.message)}
+              {errors?.confirmPassword &&
+                errors?.confirmPassword.message &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)